### PR TITLE
Gorelase fixes

### PR DESCRIPTION
**Pull request by christophercampbell**
This PR applies some of the gorelease fixes discovered on cdk-validium-node to this project

- add version command
- fix version string if necessary for docker
- apply packr2 prior to gorelease build
- adjustment to binary location in docker file for consistence & endpoint -> cmd for override

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -31,7 +37,7 @@ archives:
 
 dockers:
   - image_templates:
-      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-amd64
+      - 0xpolygon/{{ .ProjectName }}:{{ replace .Version "+" "-" }}-amd64
```
Comment (Stefan-Ethernal) [inline .goreleaser.yml:40]: Are we going to name CDK DAC releases with the `+cdk` suffix as well?

File: .goreleaser.yml [hunk 2]
```diff
@@ -6,6 +6,12 @@ release:
   draft: true
   prerelease: auto
 
+before:
+  hooks:
+    - go mod download
+    - go install github.com/gobuffalo/packr/v2/packr2@v2.8.3
+    - packr2
```
Comment (Stefan-Ethernal) [inline .goreleaser.yml:13]: Just curious:
when and if we approach with the embedded migrations, we wouldn't need this part as the pre-execution step?

**Review by Stefan-Ethernal — APPROVED** [review]

File: .goreleaser.yml (hunk 1 above)
Comment (christophercampbell) [reply .goreleaser.yml:40]: for consistency, just in case, should I remove?

File: .goreleaser.yml (hunk 2 above)
Comment (christophercampbell) [reply .goreleaser.yml:13]: that's right

File: .goreleaser.yml (hunk 1 above)
Comment (Stefan-Ethernal) [reply .goreleaser.yml:40]: Nope, good with me, I was asking more out of curiosity.

**Pull request merged**